Author: Pino Toscano <pino@debian.org>
Description: Remove files stripped from tarball
 Remove cvs.pdf (sourceless) from the build system (even if not installed).
Origin: vendor
Forwarded: not-needed
Last-Update: 2014-10-11

--- a/Makefile.am
+++ b/Makefile.am
@@ -10,7 +10,6 @@ README \
 test_csv.c \
 tests \
 examples \
-csv.pdf \
 COPYING.LESSER
 TESTS = check_csv
 check_PROGRAMS = check_csv
